tests/test_coverity.py

```diff
@@ -11,6 +11,16 @@
 TEST_OUT_DIR = Path(__file__).parent / 'test_out'
 
 
+def ordered(obj):
+    if isinstance(obj, dict):
+        return sorted((k, ordered(v)) for k, v in obj.items())
+    if isinstance(obj, list):
+        return sorted(ordered(x) for x in obj)
+    else:
+        return obj

```

Comment:
I think it would be better to update the reference file for the test outputs. The order of the CQ entries should be deterministic.

Reply:
Why should the order be deterministic?

Reply:
If someone is ever interested in a manual diff, the order is important. It shouldn't take much effort to make it deterministic.

Reply:
Resolved in 45fcc8a74499d5f924acf050bcc2bed24534fac3